CSS/index.css

```diff
@@ -280,7 +280,7 @@
   width: 100%;
   text-align: center;
   padding: 14px 0;
-  font-size: 16px;
+  font-size: 18px;
```

Comment:
Remember accessibility; it's best practice to set the font size on body to 62.5% and then element font sizes in rem. 1 rem = 10px.